DNN Platform/DotNetNuke.Abstractions/Application/SchedulerMode.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information
+
+namespace DotNetNuke.Abstractions.Application;
+
+/// <summary>The method for triggering scheduled tasks.</summary>
+public enum SchedulerMode
+{
+    /// <summary>The scheduler is disabled.</summary>
+    Disabled = 0,
+
+    /// <summary>The scheduler is running based on a timer.</summary>
+    TimerMethod = 1,
+
+    /// <inheritdoc cref="TimerMethod"/>
+    TIMER_METHOD = TimerMethod,

```

Comment:
Should these be `[DnnDeprecated]` and maybe `[EditorBrowsable(EditorBrowsableState.Never)]` (so they don't show up in Visual Studio auto-complete)?

Reply:
I like `[EditorBrowsable(EditorBrowsableState.Never)]`, I'm not sure about `DnnDeprecated` since we need the versions with underscores for parsing the value in the database.

Reply:
Does the simple [Obsolete] work in this context ?

Reply:
I've pushed an `[Obsolete]` attribute